### DISCUSSION TITLE
Installing nordvpn container does not create data directory in appdata and does not run

### DISCUSSION THREAD

**Discussion by jimdaway** [bubuntux/nordvpn#279] [Q&A]
### Discussed in https://github.com/bubuntux/nordvpn/discussions/257

<div type='discussions-op-text'>

<sup>Originally posted by **jimdaway** October 11, 2021</sup>
I certainly would appreciate some guidance here.

On unRAID 6.9.2, when I install nordvpn docker container, there is no entry/directory created in the /mnt/user/appdata folder. However the install appears to complete properly yet when I start the container, it reports invalid username or password in the log and then goes into a stopped state.

I had this installed and running on a test server a couple of days ago then tore down that server in favor of the "production" version. Now, the directory in appdata isn't being created. Other containers are installing as expected and creating a directory in the appdata folder.

I have '--device /dev/net/tun --cap-add=NET_ADMIN" in the Extra Parameters field, my network type is set to Bridge and I've confirmed that the username/password is entered properly.

Finally, my appdata share is set to Public and Prefer : cache
![Capture](https://user-images.githubusercontent.com/5943652/136842982-99ef97e1-1839-448a-b43a-502435716cde.JPG)

</div>

**Comment by ThxAndBye** [comment]
This container doesn't need a directory in appdata on unRAID as the container doesn't need a persistent storage on its own. All the configuration is done via the configuration (environment variables, port forwards, parameters, etc.).  
I'm not sure if Wireguard (NordLynx) still requires the SYS_MODULE though, I remember when I set it up that I needed to add it.
